src/modules/market/components/market-outcomes-yes-no-scalar/market-outcomes-yes-no-scalar.jsx

```diff
@@ -61,10 +44,23 @@ const MarketOutcomes = p => {
     p.type !== YES_NO
       ? ""
       : getValue(p.outcomes[0], "lastPricePercent.denomination");
-  const arrowStyles = {
-    left: p.type === YES_NO ? "0.9375rem" : "0.5rem",
-    marginLeft: p.type === YES_NO ? "0.625rem" : "0",
-    marginBottom: -5
+
+  const currentMarketStyles = pos => {
+    let size =
+      getValue(p.outcomes[0], "lastPricePercent.formatted").toString().length *
+      0.9;
+    const isMobileAttrs =
+      window.outerWidth < 590
+        ? { marginTop: 4, fontSize: 16 }
+        : { marginTop: 7, fontSize: 20 };
+    for (let i = 6; i < size; i += 6) {
+      size *= 0.7;
+    }
+    return {
+      marginLeft: pos < 15 ? size + "em" : 0,
+      marginRight: pos > 85 ? size + "em" : 0,
```

Comment:
Can these be changed to use rem instead of em? I believe we're only supposed to be using rem.

Reply:
done!